examples/chart/teleport-demo/docker/build-all.sh

```diff
@@ -1,15 +1,25 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -e
+ONE=$1
```

Comment:
You're setting `$ONE` here but you don't actually use it. Probably best to just not set it in that case.